projects/snapshot_bot/snapshot_bot.cc

```diff
@@ -259,16 +261,18 @@ class RobotFSM : FSM<State>::StateHandler
                     }
 
                     const double elapsed = static_cast<millisecond_t>(m_RecordingStopwatch.elapsed()).value();
-
                     // If Vicon tracking is available
                     if(m_Config.shouldUseViconTracking()) {
 #ifdef USE_VICON
                         // Get tracking data
                         const auto objectData = m_ViconTracking.getObjectData(m_Config.getViconTrackingObjectName());
+                        const auto imuData = imu.getEulerAngles();
```

Comment:
It would be clearer if this was ``const std::array<degree_t, 3> imuData = ...``. You want to be explicit about the fact that these values are degrees so we know that we're not accidentally saving values in radians.